planning/motion_velocity_smoother/include/motion_velocity_smoother/motion_velocity_smoother_node.hpp

```diff
@@ -130,6 +133,8 @@ class MotionVelocitySmootherNode : public rclcpp::Node
 
   double over_stop_velocity_warn_thr_;  // threshold to publish over velocity warn
 
+  mutable rclcpp::Clock::SharedPtr clock_;
```

Comment:
This is to get rid of the local clock definition.
https://github.com/tier4/autoware.universe/blob/tier4/main/planning/motion_velocity_smoother/src/motion_velocity_smoother_node.cpp#L430

Because to call a clock is not a const function, the local clock is defined to make the function const. But there is no way to make a local clock with `RCL_ROS_TIME` even with the following decleration. 
My solution is to make a mutable clock and use it.

```
clock{rclcpp::Clock{RCL_ROS_TIME}};
```